src/oscar/apps/dashboard/vouchers/views.py

```diff
@@ -2,10 +2,11 @@
 
 from django.contrib import messages
 from django.core.urlresolvers import reverse
-from oscar.core.loading import get_model
 from django.http import HttpResponseRedirect
+from django.utils import timezone
 from django.utils.translation import ugettext_lazy as _
 from django.views import generic
+from oscar.core.loading import get_model
```

Comment:
@mvantellingen Of course! I'll group the includes next week, when I'll back from vacation.